packages/client/components/TeamPrompt/TeamPromptLastUpdatedTime.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {MenuPosition} from '../../hooks/useCoords'
+import useTooltip from '../../hooks/useTooltip'
+import {PALETTE} from '../../styles/paletteV3'
+import absoluteDate from '../../utils/date/absoluteDate'
+import relativeDate from '../../utils/date/relativeDate'
+
+const Timestamp = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontWeight: 600,
+  fontSize: 12
+})
+const Hover = styled('span')({
+  cursor: 'pointer'
+})
+
+interface Props {
+  createdAt: string | Date
+  updatedAt: string | Date
+}
+
+export default function LastUpdatedTime({updatedAt, createdAt}: Props) {
+  const {
+    tooltipPortal: createdTimePortal,
+    openTooltip: showCreatedTime,
+    closeTooltip: closeCreatedTime,
+    originRef: createdTimeRef
+  } = useTooltip<HTMLButtonElement>(MenuPosition.UPPER_CENTER)
+  const {
+    tooltipPortal: updatedTimePortal,
+    openTooltip: showUpdatedTime,
+    closeTooltip: closeUpdatedTime,
+    originRef: updatedTimeRef
+  } = useTooltip<HTMLButtonElement>(MenuPosition.UPPER_CENTER)
+
+  const isEdited = createdAt !== updatedAt
+  return (
+    <Timestamp>
+      <Hover onMouseEnter={showCreatedTime} onMouseLeave={closeCreatedTime} ref={createdTimeRef}>
```

Comment:
I see what you mean, it's because the hover triggers a re-render of the component, so there's a jump from "just now" to "3 minutes ago", but do you expect this time to keep updating automatically? That might require a poll, is that too heavy or do you have other suggestions for it?

Reply:
We don't need to poll the server, we would just need to reformat the time. Ideally this would be done outside of React component tree by updating the ref via script directly, i.e. add a setInterval which takes the ref of the text component and updates it without doing a full re-render. @nickoferrall I think you have more React experience, what do you think?

Reply:
Yep, I'd add a setInterval to a useEffect and update the timestamp that way. Interestingly, in our Discussion Thread, I noticed we're not updating the timestamp until the component rerenders, so if you implement a pattern in standups, it could be useful elsewhere in the app.